documentation/adr/0002-use-github-for-documentation.md

```diff
@@ -16,14 +18,21 @@ Documentation will directly reference Horace and Herbert projects.
 
 The GitHub Wiki repository does not support pull requests, or the display of any content not on the the `master` branch.
 
+
 ## Decision
 
 We will store documentation in the `Horace/documentation` folder and link to this from the the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub. 
```

Comment:
@oerc0122 Presumably the folder in this project, so add "folder in the Horace Github repository."